src/main/java/net/adoptium/documentationservices/services/RepoService.java

```diff
@@ -60,6 +70,41 @@ public boolean isUpdateAvailable() throws IOException {
         return repoLastUpdated.isAfter(lastUpdateTimestamp);
     }
 
+    public Set<Contributor> getContributors(final Documentation documentation) throws IOException {
+        final Set<GHUser> commitAuthors = new HashSet<>();
+        final GHRepository repo = getGitHubRepo();
+
+        // iterate over all files for given documentation, that way we won't miss contributors of e.g. images.
+        final List<GHContent> documentationContents = getGitHubRepo().getDirectoryContent(documentation.getId());
+        documentationContents.stream().filter(ghContent -> ghContent.isFile()).forEach(ghContent -> {
+            // retrieve commits for file and extract author
+            final GHCommitQueryBuilder commitQueryBuilder = repo.queryCommits();
+            PagedIterable<GHCommit> commits = commitQueryBuilder.path(ghContent.getPath()).list();
+            commits.forEach(ghCommit -> {
+                try {
+                    final GHUser author = ghCommit.getAuthor();
+                    if (author != null) {
+                        commitAuthors.add(author);
+                    }
+                } catch (IOException ioe) {
+                    LOG.error("Failed to retrieve author of commit {}.", ghCommit.getSHA1());
```

Comment:
```suggestion
                    LOG.error("Failed to retrieve author of commit " + ghCommit.getSHA1(), ioe);
```
by doing so we will log the stack trace